src/components/TileGallery/TileGallery.story.jsx

```diff
@@ -0,0 +1,429 @@
+/* Used dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import Check from '@carbon/icons-react/lib/checkmark--filled/16';
+import Rocket from '@carbon/icons-react/lib/rocket/32';
+import IconStarFav from '@carbon/icons-react/lib/star--filled/16';
+import Activity from '@carbon/icons-react/lib/activity/32';
+import Sunny from '@carbon/icons-react/lib/sunny/32';
+
+import {
+  PageTitleBar,
+  TileGallery,
+  TileGallerySection,
+  TileGalleryItem,
+  TileGalleryViewSwitcher,
+  StatefulTileGallery,
+  Link,
+  OverflowMenu,
+  OverflowMenuItem,
+} from '../../index';
+import FullWidthWrapper from '../../internal/FullWidthWrapper';
+
+export const content = (
+  <TileGallery>
+    <TileGallerySection title="Section 1">
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Rocket fill="black" description="Icon" width={50} height={50} />}
+        icon={<Check fill="#40be65" onClick={() => alert('favorite')} />}
+        afterContent={
+          <OverflowMenu
+            floatingMenu
+            onClick={evt => evt.preventDefault()}
+            style={{ height: '2rem' }}
+          >
+            <OverflowMenuItem
+              itemText="Set as home page"
+              onClick={() => alert('Set as home page')}
+            />
+          </OverflowMenu>
+        }
+      />
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Activity fill="black" description="Icon" width={50} height={50} />}
+        icon={<Check fill="#40be65" onClick={() => alert('favorite')} />}
```

Comment:
Color variables should be used, instead of hardcoded hex values

Reply:
This should be fine for the story I imagine.